canvas_manage_course/settings/base.py

```diff
@@ -48,20 +50,20 @@
     'lti_school_permissions',
     'manage_people',
     'manage_sections',
-)
+]
 
-MIDDLEWARE_CLASSES = (
+MIDDLEWARE_CLASSES = [
```

Comment:
We should change this to MIDDLEWARE as MIDDLEWARE_CLASSES is deprecated. 
https://docs.djangoproject.com/en/1.10/ref/settings/#std:setting-MIDDLEWARE

I wonder if changing this will mitigate the need to do an explicit import of the patched method. Could you give this a try and if it works also apply it to CAAT?

Reply:
Good catch! Unfortunately it does not resolve that issue. I currently am using MIDDLEWARE in CAAT and forgot to make the same here

Reply:
Ah ok! was hoping that was it :-)